Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/Runner.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.cdp.volha_ivanova.task1;
+
+import com.epam.cdp.volha_ivanova.task1.sweets.Gift;
+
+public class Runner {
+	/**
+	 * @author Volha_Ivanova 
+	 * Implements calculating weight of the present according to number of sweets that enter user. Sort sweets according to number in the present. Find sweet weight by its name.
+	 */
+	public static void main(String[] args) {
+		System.out.println("How many \n waffles \n white chocolates \n dark chocolates \n milk chocolates \n chocolate candies \n lollipops \n do you want to add to the present?");
```

Comment:
It's a really long message. Please use StringBuilder to append parts of the message and  use System.lineSeparator() instead of \n